packages/flutter/lib/src/widgets/default_text_editing_shortcuts.dart

```diff
@@ -431,4 +427,32 @@ class DefaultTextEditingShortcuts extends Shortcuts {
         return _windowsShortcuts;
     }
   }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget result = child;
+    if (kIsWeb) {
+      // On web platform, these shortcuts makes sure the following:
+      //
+      // 1. Shortcuts fired when a TextField is focused are ignored and
+      //    forwarded to the browser.
+      // 2. Shortcuts fired when no TextField is focused will still trigger
+      //    default text editing Shortcuts
+      //
+      // In (2), the disabling shortcuts will still fire; however, the
+      // [DoNothingAndStopPropagationTextIntent] will not be handled because the
+      // context is above the TextField. The key event will then trigger the
+      // default Text Editing Shortcuts.
```

Comment:
Once the above changes have been made to 1 and 2, remove this paragraph?

Reply:
I think the docs here will be good with the changes above, but it has the effect of coupling DefaultTextEditingShortcuts and EditableText.  Maybe that's not a big deal because the coupling is kind of inherent.  Or we could be generic in the comments above to remove the coupling, but maybe it's not worth it?

Reply:
I think this should be fine unless we would have other text editing widget that do no use EditableText.